reportportal_client/model/response.py

```diff
@@ -1,12 +1,15 @@
 import json
 
+from ..errors import EntryCreatedError, OperationCompletionError
+
 
 class RS(object):
     def __init__(self, raw):
         super(RS, self).__init__()
         self.raw = raw
 
 
+# FIXME: need to improve and rework errors handling, add checks of err codes
```

Comment:
Please add new issue for this part of coffee, i think it's better than add comment to code